README.md

```diff
@@ -58,9 +58,49 @@ Do either of the following:
 -   Move them to `/usr/local/lib` or `/usr/lib`.
 -   Set the `LIBRARY_PATH` environment variable to the directory containing the libraries.
 
-### Execution
+### Compilation (static libraries)
+
+For iOS development you have to use static libraries. Get the static libraries from the [vosk-api][vosk-api-ios] team.
+
+#### macOS-only
+The Rust linker is not able to extract the correct architecture from a static fat file itself. Therefore, for each architecture we need a non-fat file to link with our code.
+
+Example for aarch64:
+```shell
+lipo -info libvosk.a
+# Architectures in the fat file: libvosk.a are: armv7 armv7s arm64 
+lipo libvosk.a -thin arm64 -output arm64/libvosk.a
+lipo -info arm64/libvosk.a
+# Non-fat file: arm64/libvosk.a is architecture: arm64
+```
+A typical use case is to create a lib using Rust to integrate with an iOS App. So the next step is to add `staticlib` as crate-type in the `Cargo.toml` file.
```

Comment:
I think this doesn't belong in the README. You're very welcome to also contribute an example project to the crate that shows a full android+iOS project (the more info, the better!), but imo specifying what compiler flags to use (like in the bullet point in my previous comment) suffices here.